lib/matplotlib/backends/backend_qt4agg.py

```diff
@@ -72,16 +45,6 @@ class FigureCanvasQTAgg(FigureCanvasQTAggBase,
 
     """
 
-    def __init__(self, figure):
-        if DEBUG:
-            print('FigureCanvasQtAgg: ', figure)
-        FigureCanvasQT.__init__(self, figure)
```

Comment:
I think this needs to be here or the MI does not work correctly (as pyqt5 made them cooperative, but they are not int pyqt4)